MIGRATING-0.6-0.7.md

```diff
@@ -162,6 +163,9 @@ To reduce duplication, please search the `DrawTarget` documentation on <https://
 
 The `Drawable` trait now uses an associated type for its pixel color instead of a type parameters.
 
+An associated type, `Output`, has also been added which can be used to return intermediate values
+from drawing operations. To maintain compatibility with 0.6, this can be set to `type Output = ();`
```

Comment:
Is "compatibility with 0.6" the right way to put this? The source code won't be compatible with e-g 0.6 anymore if the additional type is added. IMO it would be enough to mention that `()` can be used as the `Output` type if the `draw` method doesn't need to return any value.